src/backend/Csrs.Api/Features/Accounts/NewAccountAndFile.cs

```diff
@@ -72,16 +73,24 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
 
                 var bceidScope = _logger.AddBCeIdGuid(userId);
 
-                var dynamicsParty = request.Applicant.ToDynamicsModel();
-                
+                var dynamicsParty = await request.Applicant.ToDynamicsModelAsync(_dynamicsClient, cancellationToken);
+
                 // find to see if the person has an account already?
                 string partyId = await _dynamicsClient.GetPartyIdByBCeIdAsync(userId, cancellationToken);
                 
                 if (partyId is not null)
                 {
                     _logger.LogDebug("Party already exists");
                     dynamicsParty.SsgCsrspartyid = partyId;
-                    await _dynamicsClient.Ssgcsrsparties.UpdateAsync(partyId, dynamicsParty, cancellationToken);
+
+                    try
+                    {
+                        await _dynamicsClient.Ssgcsrsparties.UpdateAsync(partyId, dynamicsParty, cancellationToken);
+                    }
+                    catch (Exception ex)
```

Comment:
This needs to be better error handling. What kind of exceptions could be thrown?  **We just continue processing even if there is error**?  The exception message could be better. Log the party id that had a problem. Better to have a message that will make sense if someone is just reading the splunk logs.

Reply:
Error handling was updated by reasonable message